templates/timezone.j2

```diff
@@ -1 +0,0 @@
-{{ ntp_timezone }}
```

Comment:
Is this template used anywhere?

Reply:
no, according to history it was used in `main.yml` for Debian at some point. This is why I propose to remove it.

Reply:
Okay thanks!